sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeActionsOperation.cs

```diff
@@ -152,7 +152,7 @@ public AnalyzeActionsOperation(string operationId, TextAnalyticsClient client)
 
             Id = operationId;
             //_serviceClient = client._serviceRestClient;
-            _diagnostics = client._clientDiagnostics;
+            _diagnostics = client._serviceClient.Diagnostics;
```

Comment:
When we look at the LROs, I'd like to look deeper here and see if we can't remove external access to underscore members in favor of an abstraction.